package.json

```diff
@@ -37,7 +37,7 @@
     "typescript": "5.7.2",
     "typescript-eslint": "^8.18.0",
     "undici-types": "^7.0.0",
-    "web-ext": "8.3.0",
+    "web-ext": "7.12.0",
```

Comment:
Sorry but could you tell me the reason we have to downgrade web-ext?

Reply:
Yeah, our build server currently uses nodejs v16 which doesn't has built-in `Blob` support, but it's needed for `web-ext v8`. So I've temporarily downgraded it to v7 for current release.

Before next release I'll check if we should use some `blob-polyfill` or maybe we'll update node on build server to the newer version.

Reply:
Aha, i see
Thank you for your explanation.